Neos.Flow/Classes/I18n/Xliff/XliffParser.php

```diff
@@ -28,67 +26,26 @@
  * @throws Exception\InvalidXliffDataException
  * @see http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html [1]
  * @see http://docs.oasis-open.org/xliff/v1.2/xliff-profile-po/xliff-profile-po-1.2-cd02.html#s.detailed_mapping.tu [2]
+ * @deprecated use \Neos\Flow\I18n\Xliff\V12\XliffParser instead. Will be removed in Flow 5.0
```

Comment:
AFAIK it's ok to remove in 5.0, there is still a LTS before that, so there should be no problem for users to avoid this deprecation